tests/spec/integration_spec.lua

```
@@ -0,0 +1,454 @@
+    getcwd = function() return "/test/project" end,
+    filereadable = function(path) return 1 end,
+    readfile = function(path)
+      if path:find("config") then
+        return {"return { debug = true, version = '1.0' }"}
+      elseif path:find("utils") then
+        return {"local function calculate(x) return x * 2 end", "return { calculate = calculate }"}
+      end
+      return {}
+    end,
+    writefile = function(lines, path) return 0 end,
+    jobstart = function(cmd, opts)
```

Luacheck (1.2.0)

[warning] unused argument 'cmd'

(W212)